configuration-example.yaml

```diff
@@ -44,40 +44,19 @@ authentication:
     password: password
 
 api:
-  endpointUri: https://api.oregonstate.edu/v1/web-api-skeleton/
+  endpointUri: https://api.oregonstate.edu/v1/
```

Comment:
The endpointUri will need to include `beaverbus` or the self links will be wrong. Or we can just implement that logic into the api so we don't need to maintain the endpoint name `beaverbus` in two places? Up to you.

Reply:
I think endpointUri usually isn't expected to contain the api name in our other apis, so I'll change things to be consistent with that.